crates/wit-parser/src/ast.rs

```diff
@@ -462,10 +465,31 @@ struct Stream<'a> {
 
 pub struct Value<'a> {
     docs: Docs<'a>,
-    name: Id<'a>,
+    name: VariableId<'a>,
     kind: ValueKind<'a>,
 }
 
+impl<'a> Value<'a> {
+    pub(crate) fn name(&self) -> &'a str {
+        match &self.name {
+            VariableId::Id(id) => id.name,
+            VariableId::Wildcard(_) => "*",
+        }
+    }
```

Comment:
I think it might be best to leave this out as an accessor since it's probably best for most consumers to do a `match` on this and have them determine how to handle it (unless it's super frequently called, but I'm not sure that it is)